### PR TITLE
Add validation for max fund balance value

### LINKED ISSUE #223: Entering large fund values causes http500 errrors
When supplying a large value (exceeding psql 4b int range) in "Balance" field on "Editing Fund", the server responds with http500 error. This basically limits funds value to 214748.3647 of currency. This might limit scalability of our system.
pb

### PR DESCRIPTION
**Pull request by mkasztelnik**
Fix #223

@dice-cyfronet/atmo-dev-team, @mpawlik please take a look

### CODE REVIEW

File: app/models/atmosphere/fund.rb [hunk 1]
```diff
@@ -48,7 +48,10 @@ class Fund < ActiveRecord::Base
               presence: true
 
     validates :balance,
-              numericality: true
+              numericality: {
+                only_integer: true,
+                less_than_or_equal_to: 2147483647
```
Comment (nowakowski) [inline app/models/atmosphere/fund.rb]: Huh...? :astonished:

Comment (mkasztelnik) [inline app/models/atmosphere/fund.rb]: Maximum value for an integer, which is used for this field in postgresql.

Comment (mkasztelnik) [inline app/models/atmosphere/fund.rb]: I can convert it into double, but right now I believe that 2147483647 is big enough :smile:

Comment (nowakowski) [inline app/models/atmosphere/fund.rb]: Oh, okay. Who knew? :)

**Comment by nowakowski** [comment]
:+1:

File: app/models/atmosphere/fund.rb (hunk 1 above)
Comment (mkasztelnik) [inline app/models/atmosphere/fund.rb]: http://www.postgresql.org/docs/9.1/static/datatype-numeric.html :)

**Pull request merged**